CHANGELOG.md

```diff
@@ -7,7 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please maintain both this file AND the README.md file.
 
 ## [Unreleased]
```

Comment:
Some more details (even if only very brief) about the contents of this PR, should be added to this file.

Reply:
I added very detailed log to #51 that contains all the changes introduced this summer.

Reply:
👍